clang/include/clang/AST/PreorderAST.h

```diff
@@ -27,12 +27,20 @@ namespace clang {
 
   class Node {
   public:
-    enum class NodeKind { OperatorNode, LeafExprNode };
+    // Nodes with two different kinds are sorted according to the order in
+    // which their kinds appear in this enum.
+    enum class NodeKind {
+      OperatorNode,
```

Comment:
Maybe we should rename `OperatorNode` to `BinaryOperatorNode` since we now have a separate `UnaryOperatorNode`.